dashboard/app/views/scripts/show.html.haml

```diff
@@ -5,6 +5,13 @@
   = render partial: 'layouts/race_interstitial'
 
 = render partial: 'shared/check_admin'
+
+- if @script.course_link
+  #script-breadcrumb
+    %a{href: @script.course_link}
+      %i.fa.fa-chevron-left
+      %span
+        = I18n.t('view_all_units')
```

Comment:
This would have been perhaps ideally done in React, but it's (a) a pretty simple set of markup and (b) not a part of the page that is currently rendered by React